### PR TITLE
review config display & add new options for remove delete button for …

### PR DESCRIPTION
**Pull request by orthagh**
…all actors

### CODE REVIEW

**Comment by orthagh** [comment]
image

File: hook.php [hunk 1]
```diff
@@ -71,9 +71,13 @@ function plugin_escalade_install() {
          `reassign_tech_from_cat`                  INT(11) NOT NULL,
          `cloneandlink_ticket`                     INT(11) NOT NULL,
          `close_linkedtickets`                     INT(11) NOT NULL,
-         `use_assign_user_group`                   INT(11) NOT NULL,
```
Comment (trasher) [inline hook.php]: I do not understand why those ones have been removed... There are stillr eferences to them in the code, isn't it?

File: hook.php [hunk 2]
```diff
@@ -231,6 +235,50 @@ function plugin_escalade_install() {
...
+      $migration->addField('glpi_plugin_escalade_configs', 'remove_delete_watcher_user_btn',
+                           'bool', ['value' => 1]);
+   }
+   if (! FieldExists('glpi_plugin_escalade_configs', 'remove_delete_watcher_group_btn')) {
+      $migration->addField('glpi_plugin_escalade_configs', 'remove_delete_watcher_group_btn',
+                           'bool', ['value' => 1]);
+   }
+
+   // migrate old fields
+   if (FieldExists('glpi_plugin_escalade_configs', 'remove_delete_user_btn')) {
```
Comment (trasher) [inline hook.php]: Sounds like the existing fields has not been changed in tables creation (same for the 2 foloowing fields).

**Comment by orthagh** [comment]
@trasher changes have been submitted (i think, it's kind of old), do you change your review status ?

**Pull request merged**